docs/general/whats-new.md

```diff
@@ -60,9 +60,9 @@
 
 ## [v1.4.3](https://github.com/project-zot/zot/releases/tag/v1.4.3)
 
-### Support for [OCI Artifacts and References](https://github.com/opencontainers/image-spec/blob/main/artifact.md)
+### Support for [OCI Artifacts and References](https://github.com/opencontainers/artifacts)
```

Comment:
This repo described an experimented which did not end in the current version of the spec, and is not relevant for current zot behavior.
OCI artifact support is added in zot 2.0.0, not 1.4.3, and it implements https://github.com/opencontainers/image-spec/blob/main/manifest.md#guidelines-for-artifact-usage

This is part of the `Updated OCI support` section.

Reply:
Added to Updated OCI support section in What's New in v2.0, removed from v1.4 section.

Reply:
Please fix the link as well. The repo you are pointing to is no longer relevant for the oci spec and for zot implementation.

Reply:
Fixed the link